lib/model/QubitTerm.php

```diff
@@ -336,7 +337,24 @@ public function delete($connection = null)
 
     QubitSearch::getInstance()->delete($this);
 
-    parent::delete($connection);
+    // Replicate parent class delete functionality, skipping nested set updating
```

Comment:
Is this a very common feature we'll need? I'm pretty uneasy about this change since it seems we're explicitly skipping over BaseTerm::delete() and going straight to QubitObject::delete(). BaseTerm::delete might not do much right now, but in the future if functionality needs to be added to it it won't be expected at all that our QubitTerm is bypassing it.

If this isn't a common feature we'll need I suggest just stashing a .patch somewhere for future reference if it comes up again.